patches/remove_load_extensions.patch

```diff
@@ -0,0 +1,11 @@
+--- spatialite/src/stored_procedures/stored_procedures.c.orig	2025-02-07 13:53:04
++++ spatialite/src/stored_procedures/stored_procedures.c	2025-02-07 13:53:09
+@@ -2062,7 +2062,7 @@
+ 	  return NULL;
+       }
+ #ifndef LOADABLE_EXTENSION
+-    sqlite3_enable_load_extension (handle, 1);
++    //    sqlite3_enable_load_extension (handle, 1);
```

Comment:
The system version of sqlite3 does not have support for this, so we have to disable this to prevent it being in the symbol table and screwing up linking. This does not seem to be configurable separately, as we need to have `LOADABLE_EXTENSION` in a couple of other places.